projects/pgai/pgai/vectorizer/embedders/openai.py

```diff
@@ -69,9 +70,9 @@ def _embedder(self) -> resources.AsyncEmbeddings:
     def _max_chunks_per_batch(self) -> int:
         return 2048
 
-    async def call_embed_api(self, documents: list[TokenDocument]) -> EmbeddingResponse:
+    async def call_embed_api(self, documents: list[Document]) -> EmbeddingResponse:
         response = await self._embedder.create(
-            input=documents,
+            input=cast(list[str] | Iterable[Iterable[int]], documents),
```

Comment:
:/ I'm meeeelting. Thinking on adding an ignore comment here instead 😓

Reply:
The typing is subtly incorrect for this whole thing, and is driving me mad (I was the one who added the incorrect stuff). I think that we can merge this as-is though.